sdk/js/package.json

```diff
@@ -28,7 +28,9 @@
     "prepublishOnly": "echo \"disabled: npm test && npm run lint\"",
     "preversion": "npm run lint",
     "version": "npm run format && git add -A src",
-    "postversion": "git push && git push --tags"
+    "postversion": "git push && git push --tags",
+    "relayer-status": "npm run test-relayer-testnet -- ./src/relayer/__tests__/wormhole_relayer.ts -t 'Checks the status of a message'",
```

Comment:
I'm a little confused about the purpose of these npm scripts. Are they to run tests? If so, should they be prefixed with `test`? Maybe I'm missing something